src/js/lib/helpers.js

```diff
@@ -348,6 +348,37 @@
 		}
 	};
 
+		/**
+	 * Attempt to get a value from localStorage
+	 *
+	 * @param string key
+	 * @return string The value obtained from localStorage, or
+	 *                undefined if localStorage is inaccessible
```

Comment:
This documentation says it returns undefined, but actually you return null.  Does it matter?

Reply:
Tidied this up to be consistent with the localStorage methods.